src/fides/api/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -1749,4 +1759,4 @@ def _process_privacy_request_restart(
         from_step=failed_step.value,
     )
 
-    return privacy_request
+    return privacy_request  # type: ignore
```

Comment:
instead of blanket ignores, can you add specific ignore rules here like you did elsewhere? Makes it a lot easier to understand _why_ it is ignored and potentially fix later